Source/Core/Common/SettingsHandler.cpp

```diff
@@ -9,9 +9,10 @@
 #include <cstddef>
 #include <ctime>
 #include <iomanip>
-#include <sstream>
 #include <string>
 
+#include <fmt/time.h>
```

Comment:
Not sure if you noticed, but `time.h` had been removed in `fmt` master (over here: fmtlib/fmt@b97e5d8c2ac858ae563c7573c4dd79d237565f99) as it was merged into `chrono.h` (over here: fmtlib/fmt@a939c75956d27996680ce498529da0d1668c4eac)

Doesn't affect us too much at this point, but is a breaking change if/when we update `fmt`.

Reply:
Will also be a problem on Linux if #8198 gets merged .

Reply:
The most recent _release_ is still 5.3.0 (which we have in Externals), so this isn't a problem until their next release.